rebar.config

```diff
@@ -12,5 +12,5 @@
 ]}.
 
 {deps, [
-  {cuttlefish, ".*", {git, "git://github.com/basho/cuttlefish.git", {tag, "2.0.5"}}}
+  {cuttlefish, "2.0.7"}
```

Comment:
I think it's 2.0.6 made to work with rebar3/hex but @tsloughter can answer that question better he made the cuttlefish package